### DISCUSSION TITLE
Asking for more examples for Shiny in quarto documents

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#1115] [Feature Requests]
The quarto documentation is excellent.  I would like to ask very kindly if is it posiible to include more complex Shiny examples or point it out to other examples.  I found that Shiny is much easier in quarto than in an rmarkdown document.

**Comment by jjallaire** [comment]
Thanks for the suggestion! We will try to add more Shiny examples soon.

**Comment by mcanouil** [comment]
I believe the page is now better regarding examples: https://quarto.org/docs/interactive/shiny/index.html
